src/Avalonia.Controls/Primitives/RangeBase.cs

```diff
@@ -129,6 +155,14 @@ public double LargeChange
             set => SetValue(LargeChangeProperty, value);
         }
 
+        protected override void OnInitialized()
+        {
+            base.OnInitialized();
+
+            Maximum = ValidateMaximum(Maximum);
```

Comment:
I think we need to apply `Minimum` here as well.

Reply:
I don't think so - in the old code `Minimum` was never coerced, I think because you need _one_ value to not depend on the others or you get a circular dependency.